### ISSUE TITLE
Reports Sizing Issue

### ISSUE THREAD

**Issue by stellarcheese** [opensourcepos/opensourcepos#3184]
### Background information

__IMPORTANT: If you choose to ignore this issue report template, your issue will be closed as we cannot help without the requested information.__

Please make sure you tick (add an x between the square brackets with no spaces) the following check boxes:

- [] Reporting an issue of an unmodified OSPOS installation
- [] Checked [open and closed issues](https://github.com/opensourcepos/opensourcepos/issues?utf8=%E2%9C%93&q=is%3Aissue) and no similar issue was already reported (please make sure you searched!)
- [] Read [README](https://github.com/opensourcepos/opensourcepos/blob/master/README.md), [WHATS_NEW](https://github.com/opensourcepos/opensourcepos/blob/master/WHATS_NEW.txt), [INSTALL.md](https://github.com/opensourcepos/opensourcepos/blob/master/INSTALL.md) and [UPGRADE](https://github.com/opensourcepos/opensourcepos/blob/master/UPGRADE.txt)
- [] Read the [FAQ](https://github.com/opensourcepos/opensourcepos#faq) for any known install and/or upgrade gotchas (in specific PHP extensions installed)
- [] Read the [wiki](https://github.com/opensourcepos/opensourcepos/wiki)
- [] Executed any database upgrade scripts if an upgrade pre 3.0.0 (e.g. database/2.4_to_3.0.sql)
- [] Aware the installation code is in [bintray](https://bintray.com/jekkos/opensourcepos/opensourcepos/view/files?sort=updated&order=asc#files) (see README), and [GitHub master](https://github.com/opensourcepos/opensourcepos/tree/master) is for [developers only](https://github.com/opensourcepos/opensourcepos/wiki/Development-setup) and therefore not complete nor stable

### Installation information

- OSPOS version is: 3.3.4
- OSPOS git commit hash is: 
- PHP version is: 7.3.27
- MySQL or MariaDB version is:  mysqlnd 5.0.12-dev - 20150407 - $Id:
- OS and version is: windows 10
- WebServer is: Xampp
- Selected language is: enlgish
- (If applicable) Docker installation: 
- (If applicable) Installation package for the LAMP/LEMP stack is: 

### Issue / Bug  / Question / New Feature

Issue with font size on reports. Also an issue when attempting to print the reports, the graph gets cut off. Attached images to show

I managed to resize the font on labels by changing html class font size in bootstrap.min but this seems a bit too much. Feel there should be a class in the chart CSS to manage this.

I will continue to look for a better fix with it. For me having the whole title on one line would work best, not to size down the font.

The printing issue, I'm not sure how to address as I haven't worked much with print function, should have a print icon link to print the report.
![ospos item report text size error](https://user-images.githubusercontent.com/83611293/118325012-72f43d00-b4c8-11eb-8457-da471878f4d2.PNG)
![report issue](https://user-images.githubusercontent.com/83611293/118325013-74256a00-b4c8-11eb-9472-c81dad5a9545.PNG)

**Comment by jekkos** [comment]
I guess you might want to look whether the chartist library offers a solution here.

**Comment by stellarcheese** [comment]
Sorry, I run my shop solo most days so little time to work on.  What i did find is the label font size is being overridden with the html tag font-size.  Meaning the ct-text-size in chartist is overiden for anyone that doesn't understand.

Changed in bootstrap.min ||

I was able to decrease the text size in the html tag. So far I have not seen that it effects any other text within ospos

will update again when I find a solution that forces label to stretch on one line. one line so you do not have to size way done to almost unreadable. Adjusting the class inspected margins, and changing I had no results.  You can resize the graph in the container class 
![Capture](https://user-images.githubusercontent.com/83611293/119183811-367d8f80-ba3a-11eb-8cd3-f72bf927498c.PNG)

**Comment by stellarcheese** [comment]
I don't believe this is something that should be spent time adding a feature to change with-in ospos, but I will make up a wiki for how to do it for anyone in future once completed

Bringing me back to highschool when I did plugins and for phpbb   sad to see the phpbbsocialnetwork project shutdown
